getting_started.ipynb

```diff
@@ -280,7 +288,7 @@
     }
    ],
    "source": [
-    "!ls"
+    "assert(os.path.exists(filename))"
```

Comment:
For consistency, `filename` should also be used further down in: `df = read_file('demodata_geolife.gpkg')`

Reply:
There you go! :)

Reply:
Thank you!